src/System.Management.Automation/engine/CommandCompletion/PseudoParameterBinder.cs

```diff
@@ -360,8 +357,8 @@ public static StaticBindingResult BindCommand(CommandAst commandAst, bool resolv
 
             return new StaticBindingResult(commandAst, pseudoBinding);
         }
-        private static Object s_bindCommandLock = new Object();
-        private static PowerShell s_bindCommandPowerShell = null;
+        [ThreadStatic]
+        static Runspace bindCommandRunspace = null;
```

Comment:
Please stick to the existing convention of the variable naming, the variable here should be named as `s_bindCommandRunspace`